Admin/AdminHelper.php

```diff
@@ -184,18 +184,4 @@ public function addNewInstance($object, FieldDescriptionInterface $fieldDescript
 
         $object->$method($instance);
     }
-
-    /**
-     * Camelize a string.
-     *
-     * @static
-     *
-     * @param string $property
-     *
-     * @return string
-     */
-    public function camelize($property)
```

Comment:
This is BC break.

If this method is really not needed, it should be deprecated instead.